package.json

```diff
@@ -29,11 +29,10 @@
     "canjs-plugin",
     "donejs"
   ],
-  "system": {
```

Comment:
@matthewp I remember you recommending we leave these as `system` for libraries so that they can be loaded with Steal 0.16 or Steal 1.0. Should we continue doing that?